docs.html

```diff
@@ -738,6 +812,22 @@ <h2 class="drawer-title center">Title</h2>
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
remove empty space, use proper indentation. you can use a **Prettier extension** for that.

Reply:
Yes I am learning how to use prettier